packages/api/src/routes/index.ts

```diff
@@ -38,7 +37,7 @@ export default (app: Application) => {
   app.use("/user", processCxId, reportDeviceUsage, user);
 
   // medical routes with API key auth - report usage is on individual routes
-  app.use("/medical/v1", processCxId, checkMAPIAccess, medical);
+  app.use("/medical/v1", processCxId, checkMAPIAccess, medicalDash);
```

Comment:
Why are we exposing this to the API?

Reply:
not intentional this is from my dev work.

Reply:
fixed